client/src/pages/SitterDetails/SitterDetails.tsx

```diff
@@ -0,0 +1,74 @@
+import CssBaseline from '@material-ui/core/CssBaseline';
+import { Grid, Avatar, Paper, Box, Typography, Divider } from '@material-ui/core';
+import LoginForm from '../Login/LoginForm/LoginForm';
+import AuthHeader from '../../components/AuthHeader/AuthHeader';
+import LogoHeader from '../../components/LogoHeader/LogoHeader';
+import logo from '../../Images/logo.png';
+import ImageList from '@material-ui/core/ImageList';
+import ImageListItem from '@material-ui/core/ImageListItem';
+
+import useStyles from './useStyles';
+
+export default function SitterDetails(): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Grid container component="main" className={classes.root}>
+      <CssBaseline />
```

Comment:
@moffatethan mentioned before _baselines should not be included in components and only in the main parent structure._

Reply:
Removed